.kitchen_configs/kitchen.docker.yml

```diff
@@ -77,7 +84,13 @@ suites:
       custom_options: '-e "include roles_profiles::roles::gecko_t_linux_talos"'
     attributes:
     includes:
-      - ubuntu-18.04  # no moonshot 2204 config yet
+      - ubuntu-18.04
```

Comment:
I think we should have a single 'linux' suite block and override the custom options for provisioner in the platforms section like:

```
platforms:
  - name: ubuntu-18.04
  - name: ubuntu-22.04
    provisioner:
      custom_options: '-e "include roles_profiles::roles::gecko_t_linux_2204_talos"'
```

Reply:
Sure! Will edit!